portal/server/helpers/constructPayload.js

```diff
@@ -5,16 +5,24 @@
  * @param {Object} body Request body (req.body)
  * @param {Array} properties Interested properties to be added into payload
  * @param {Boolean} csrf Include CSRF token, defaulted to `true`
+ * @param {Array} unsetIfPropertyIsEmpty The condition to determine whether to delete the property
```

Comment:
Comment is a bit confusing, not sure if we can call it `condition`.

Maybe this is better `list of property names that can't have empty value as they will have object value later.`

Reply:
* @param should be above Boolean parameter.
* `propertyEmpty` as a variable name will be clear.

Reply:
- Changed the comment to List of property names that can't have empty value as they will have object value later, defaulted to true
- @param is now boolean `@param {Boolean} propertyEmpty`
- Changed the variable name to `propertyEmpty`